static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -157,6 +157,14 @@ const HISTORICAL_TIME_PERIOD_MAP: TimePeriodMap = {
   [TimePeriod.FOURTEEN_DAYS]: '42d',
 };
 
+const HISTORICAL_TIME_PERIOD_MAP_FIVE_MINS: TimePeriodMap = {
+  [TimePeriod.SIX_HOURS]: '678h',
+  [TimePeriod.ONE_DAY]: '29d',
+  [TimePeriod.THREE_DAYS]: '31d',
```

Comment:
Can DRY this up by using the existing values and overriding the problematic ones

```suggestion
  ...HISTORICAL_TIME_PERIOD_MAP,
```